komodo/link.py

```diff
@@ -0,0 +1,56 @@
+import time
+import os
+import shutil
+
+import komodo
+
+age_limit = 7200
+
+
+def update_links(prefix):
+    # The first step is to scan the prefix directory and inspect all the
+    # directories in the folder. The directory names are split, and a
+    # dictionary:
+    #
+    #     release_list = {"2018.03" : [(1234, "2018.03-1234"),
+    #                                  (5678, "2018-03-5678")],
+    #                     "2018.10" : [(561, "2018.10.561"),
+    #                                  (777, "2018.10-777")]}
+    #
```

Comment:
I would prefer the timestamp is something that would be lexicographically sortable (i.e. not a running number, but a more complex date string, or at the very least 0-padded)

Reply:
It liked the unix timestamp approach - but I have zero padded it to make it sortable in the shell.